private/rules/coursier.bzl

```diff
@@ -794,6 +794,7 @@ def make_coursier_dep_tree(
         fetch_sources,
         fetch_javadoc,
         timeout,
+        additional_options,
```

Comment:
Since this is just for `coursier`, could you please name the parameter `additional_coursier_options`?

Reply:
Since this file is dedicated to coursier, this may not be needed, right?

Reply:
NVM I think using the same name is better for readability.